x/auction/types/auctions.go

```diff
@@ -173,9 +193,10 @@ func NewDebtAuction(buyerModAccName string, bid sdk.Coin, initialLot sdk.Coin, e
 			Lot:             initialLot,
 			Bidder:          supply.NewModuleAddress(buyerModAccName), // send proceeds from the first bid to the buyer.
 			Bid:             bid,                                      // amount that the buyer is buying - doesn't change over course of auction
-			HasReceivedBids: false,                                    // new auctions don't have any bids
+			HasReceivedBids: true,                                     // new auctions don't have any bids
```

Comment:
^^

Reply:
This should be false.

The problem is here https://github.com/Kava-Labs/kava/blob/9932169a6786e2a58ef9bc5d9c4195246c9be8b0/x/auction/simulation/genesis.go#L74

It should create an auction and then manually set `HasReceivedBids` to `true`

Reply:
If I set it to false the auction won't be valid because of the following tho:

```go
if !a.HasReceivedBids && a.Bid.IsPositive() {
		return fmt.Errorf("cannot have a positive value bid when auction has a false HasReceivedBids flag value")
	}
```

Reply:
^ That check isn't quite right for DebtAuctions. See my comment above.